lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -101,6 +102,21 @@
 
                 setExtraData: function(data) {
                     return _.extend(data, this.user_details);
+                },
+
+                validateCountry: function() {
+                    if( !this.model.get('country') ){
+                        var $submissionContainer = $('.submission-error'),
+                            $errorMessageContainer = $submissionContainer.find('.message-copy'),
+                            $countryLabel = $('#user-country-title');
+
+                        $countryLabel.addClass('error');
+                        $errorMessageContainer.append(
+                            "<li>Please go to your <a href='/account/settings'>profile page</a> and provide your country of residence.</li>"
```

Comment:
Should the href be passed in from Django or is it fine to hard-code it here?
Translations needed for this copy.
Nitpick: inconsistent quotes. Single strings are used for strings in the rest of the file.